# request
duration prediction
---
# response
OrientationStamped orientation
